steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019_pass0_recon.lcsim

```diff
@@ -0,0 +1,439 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lcsim xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/lcsim/1.0/lcsim.xsd">
+    <!-- 
+      Steering file for running 2019 Reconstruction on readout data (LCIO or EVIO)
+      created:  9/24/19
+      @author Norman Graf <Norman.Graf@slac.stanford.edu>
+      @author PF <pbutti@slac.stanford.edu>
+    -->
+    <execute>
+        <driver name="EventMarkerDriver"/>
```

Comment:
This driver is depricated. Event marking is built into the event loop and controlled with -e